### PR TITLE
Add Owncast

### PR DESCRIPTION
**Pull request by gabek**

### CODE REVIEW

File: _data/projects/owncast.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://owncast.online
+tags:
+  - video
+  - streaming
+  - chat
+  - livestream
+  - go
+upforgrabs:
+  name: good first issue
+  link: https://github.com/owncast/owncast/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```
Comment (ritwik12) [inline _data/projects/owncast.yml]:
```suggestion
  link: https://github.com/owncast/owncast/labels/good%20first%20issue
```

**Review by ritwik12 — DISMISSED** [review]

File: _data/projects/owncast.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/owncast.yml]: @gabek

**Comment by gabek** [comment]
> Checking the latest changes to the pull request...
> 
> #### `_data/projects/owncast.yml` 
> The label 'good first issue' for GitHub repository 'owncast/owncast' does not match the specified `upforgrabs.link` value. Please update it to ``.

I'm afraid I'm not really sure what the bot is asking me to do :)

**Review by shiftkey — APPROVED** [review]
Thanks for the contribution @gabek!

Don't worry about that message, I've re-checked the URL and is what we should expect.

**Pull request merged**